lib/ui/widgets/notification_alert.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+
+class NotiDelete extends StatelessWidget {
```

Comment:
What’s the name ?? This is a dialog .. it should be named properlu .. NotificationDeleteDialog

Reply:
This page is not used anywhere, I forget to remove it.